src/open_inwoner/templates/pages/cases/contact_form.html

```diff
@@ -9,7 +9,7 @@ <h2 class="h2" id="contact">{% trans 'Stel een vraag' %}</h2>
     {% endblocktrans %}
 </p>
 
-{% render_form form=contact_form method="POST" id="contact-form" hxpost=hxpost_contact_action hxtarget="#cases-contact-form" extra_classes="case-contact-form" %}
+{% render_form form=contact_form method="POST" id="contact-form" hxpost=hxpost_contact_action hxtarget="#cases-contact-form" extra_classes="contact-form" %}
```

Comment:
You may like to move the. disabled styling to the .form SCSS instead of into .Contactform SCSS (?) - since here it will not work on this class as a general style.